doc/classes/GraphEdit.xml

```diff
@@ -433,11 +433,11 @@
 			</description>
 		</signal>
 		<signal name="graph_elements_linked_to_frame_request">
-			<param index="0" name="elements" type="Array" />
+			<param index="0" name="elements" type="StringName[]" />
 			<param index="1" name="frame" type="StringName" />
 			<description>
 				Emitted when one or more [GraphElement]s are dropped onto the [GraphFrame] named [param frame], when they were not previously attached to any other one.
-				[param elements] is an array of [GraphElement]s to be attached.
+				[param elements] is an array of names of [GraphElement]s to be attached.
```

Comment:
The clarification is good, but the additional "of" makes the sentence a bit more difficult to read.
```suggestion
				[param elements] contains the names of the [GraphElement]s to be attached.
```